code/game/objects/structures/bedsheet_bin.dm

```diff
@@ -75,6 +112,12 @@ LINEN BINS
 	else
 		return ..()
 
+/obj/item/bedsheet/AltClick(mob/living/user)
+	// double check the canUseTopic args to make sure it's correct
+	if(!istype(user) || !user.canUseTopic(src, be_close = TRUE, no_dexterity = TRUE, no_tk = FALSE, need_hands = !iscyborg(user)))
+		return
+	dir = dir & WEST ? EAST : WEST
```

Comment:
I'm not really a fan of this, do you think it would be better to just turn() it 180?

Reply:
Applied with https://github.com/tgstation/tgstation/pull/70576/commits/9535a90ffe959db5fab4443642954a151ba0758b